marklogic-data-hub-central/ui/src/config/data.config.js

```diff
@@ -72,6 +72,32 @@ const mapProps = {
     { key: '@proteinType', val: 'home' },
     { key: 'nutFree:name', val: 'testName1' }
   ],
+  entityTypePropertiesUpdated : [
+    { name: 'propId', type: 'int' },
+    { name: 'propName', type: 'string' },
+    {
+      name: 'items', type: 'parent-ItemType [ ]', children: [
+        { name: 'items/itemTypes', type: 'string' },
+        {
+          name: 'items/itemCategory', type: 'parent-catItem', children: [
+            { name: 'items/itemCategory/artCraft', type: 'string' },
+            { name: 'items/itemCategory/automobile', type: 'string' }
+          ]
+        }]
+    },
+    { name: 'gender', type: 'string' }
+  ],
+  sourceDataUpdated : [
+    { key: 'proteinId', val: '123EAC' },
```

Comment:
for an xml doc, first key itself should be a parent.

Reply:
This will be taken care of in a separate story.